train.py

```diff
@@ -0,0 +1,38 @@
+import csv
+import json
+
+def info_person(filename):
+    info_arr = []
+    with open(filename, newline='') as File:
+        reader = csv.reader(File, delimiter =',')
+        for row in reader:
+            info_arr.append(row)
+
+    info_dict = {}
+    for i in range(1,len(info_arr)):
+        person_arr = []
+        for j in range(1,len(info_arr[0])):
+            if info_arr[i][j] == "":
+                info_arr[i][j] = "no info"
+            person_arr.append(info_arr[i][j])
+            info_dict[info_arr[i][0]] = person_arr
+    print(info_dict)
+    info_json = json.dumps(info_dict)
+    print(info_json)
+
+info_person('train.csv')
```

Comment:
Please read about "typing" package. Python do not uses typing but it is very useful to track for users what each method/object/function takes